cardpopup.js

```diff
@@ -0,0 +1,290 @@
+        <li>${projectInfo[0].tools[1]}</li>
+        <li>${projectInfo[0].tools[2]}</li>
+      </ul>
+      <button class="popBtn show">See project</button>
+    </div>
+  </div>
+  </div>
+
+  <div class="card-body">
+  <div class="card-img swap">
+  <img src="${projectInfo[1].popupPhoto}" width="100%"/>       
+  </div>
+  <div>
+    <h2 class="card-header">${projectInfo[1].header}</h2>
+    <ul class="card-sub">
+      <li class="remove">${projectInfo[1].canopy}</li>
+      <div class="bullet-point"><img src="./images/icons/Counter.png" alt="Bullet Point"></div>
+      <li>${projectInfo[1].backend}</li>
+      <div class="bullet-point"><img src="./images/icons/Counter.png" alt="Bullet Point"></div>
+      <li>${projectInfo[1].year}</li>
+    </ul>
+    <div class="card-content">
+      <p>
+      ${projectInfo[1].content}
+      </p>
+      <ul class="tools">
+        <li>${projectInfo[1].tools[0]}</li>
+        <li>${projectInfo[1].tools[1]}</li>
+        <li>${projectInfo[1].tools[2]}</li>
+      </ul>
+      <button class="popBtn show">See project</button>
+    </div>
+  </div>
+  </div>
+
+  <div class="card-body">
+  <div class="card-img swap">
+  <img src="${projectInfo[2].popupPhoto}" width="100%"/>       
+  </div>
+  <div>
+    <h2 class="card-header">${projectInfo[2].header}</h2>
+    <ul class="card-sub">
+      <li class="remove">${projectInfo[2].canopy}</li>
+      <div class="bullet-point"><img src="./images/icons/Counter.png" alt="Bullet Point"></div>
+      <li>${projectInfo[2].backend}</li>
+      <div class="bullet-point"><img src="./images/icons/Counter.png" alt="Bullet Point"></div>
+      <li>${projectInfo[2].year}</li>
+    </ul>
+    <div class="card-content">
+      <p>
+      ${projectInfo[1].content}
+      </p>
+      <ul class="tools">
+        <li>${projectInfo[2].tools[0]}</li>
+        <li>${projectInfo[2].tools[1]}</li>
+        <li>${projectInfo[2].tools[2]}</li>
+      </ul>
+      <button class="popBtn show">See project</button>
+    </div>
+  </div>
+  </div>
+
+  <div class="card-body">
+  <div class="card-img swap">
+  <img src="${projectInfo[3].popupPhoto}" width="100%"/>       
+  </div>
+  <div>
+    <h2 class="card-header">${projectInfo[3].header}</h2>
+    <ul class="card-sub">
+      <li class="remove">${projectInfo[1].canopy}</li>
+      <div class="bullet-point"><img src="./images/icons/Counter.png" alt="Bullet Point"></div>
+      <li>${projectInfo[3].backend}</li>
+      <div class="bullet-point"><img src="./images/icons/Counter.png" alt="Bullet Point"></div>
+      <li>${projectInfo[3].year}</li>
+    </ul>
+    <div class="card-content">
+      <p>
+      ${projectInfo[3].content}
+      </p>
+      <ul class="tools">
+        <li>${projectInfo[3].tools[0]}</li>
+        <li>${projectInfo[3].tools[1]}</li>
+        <li>${projectInfo[3].tools[2]}</li>
+      </ul>
+      <button class="popBtn show">See project</button>
+    </div>
+  </div>
+  </div>
+  `;
```

Comment:
[OPTIONAL] First of all, great job on your code! You've done an excellent job of dynamically creating HTML elements using template literals and object properties. 💯  However, I believe that this could be made a lot easier with a loop. ➿ 

Instead of manually creating the HTML for each project, you could use a loop to iterate over the `projectInfo` array and dynamically create the HTML elements based on each object in the array. This would make your code much more concise and easier to read and maintain.

For example, you could use a `forEach` loop to create the HTML for each project like this:
```
projectInfo.forEach((project) => {
  projects.innerHTML += `
    <div class="card-body">
      <div class="card-img swap">
        <img src="${project.popupPhoto}" width="100%"/>       
      </div>
      <div>
        <h2 class="card-header">${project.header}</h2>
        <ul class="card-sub">
          <li class="remove">${project.canopy}</li>
          <div class="bullet-point"><img src="./images/icons/Counter.png" alt="Bullet Point"></div>
          <li>${project.backend}</li>
          <div class="bullet-point"><img src="./images/icons/Counter.png" alt="Bullet Point"></div>
          <li>${project.year}</li>
        </ul>
        <div class="card-content">
          <p>
            ${project.content}
          </p>
          <ul class="tools">
            <li>${project.tools[0]}</li>
            <li>${project.tools[1]}</li>
            <li>${project.tools[2]}</li>
          </ul>
          <button class="popBtn show">See project</button>
        </div>
      </div>
    </div>
  `;
});
```
This code will iterate over each object in the `projectInfo` array and create the necessary HTML elements for each project. It's a lot shorter and more readable than manually creating the HTML for each project. 📦 

I hope this helps! Let me know if you have any questions. 🤓 ❓

Reply:
Hello @didierganthier, thank you so much for reviewing my project. Also very appreciative for the fact that you went above and beyond to make the suggestions. I am working on integrating them to make my work better. :100: